src/client/camera.cpp

```diff
@@ -664,7 +664,7 @@ void Camera::wield(const ItemStack &item)
 void Camera::drawWieldedTool(irr::core::matrix4* translation)
 {
 	// Clear Z buffer so that the wielded tool stays in front of world geometry
-	m_wieldmgr->getVideoDriver()->clearZBuffer();
+	m_wieldmgr->getVideoDriver()->clearBuffers(video::ECBF_DEPTH, video::SColor(255,0,0,0), 1.f, 0);
```

Comment:
since those are the defaults no need to type them out
```suggestion
	m_wieldmgr->getVideoDriver()->clearBuffers(video::ECBF_DEPTH);
```